src/content/docs/synthetics/synthetic-monitoring/scripting-monitors/write-synthetic-api-tests.mdx

```diff
@@ -42,9 +42,9 @@ Result timing details will be provided as long as you use the `$http` object. Fo
 </Callout>
 
 <img
-  title="api-test-snap.png"
-  alt="api-test-snap.png"
-  src={apiTestSnap0}
+  title="syntheticApiTestScript"
+  alt="syntheticApiTestScript"
+  src={synthetic_screenshot-crop_api-test-script}
```

Comment:
@akristen This looks weird! It looks like it dropped the file name instead of the alias `syntheticApiTestScript` here!

Reply:
HAH